src/components/site/SiteStatus.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const SiteStatus = props => <p><span className="dg_status-text">{props.status}</span></p>;
```

Comment:
Just noticed this, you can probably just make this a span and display it as block.

Reply:
So played with this and it seems like leaving it as a <p> and removing the span worked but adding block to the css and removing the <p> broke the display